ui/appui-react/src/appui-react/widgets/WidgetDef.tsx

```diff
@@ -149,6 +149,7 @@ export class WidgetDef {
   private _defaultFloatingPosition: PointProps | undefined;
 
   private _hideWithUiWhenFloating?: boolean;
+  private _allowedPanelTargets?: PanelSide[];
```

Comment:
`PanelSide` type is @internal from `appui-layout-react`...

Reply:
We are already using PanelSide in TabLocation in WidgetDef.tsx.

Reply:
Yes, but it is marked as **@internal**, while `WidgetDef.allowedPanelTargets` is not.

Reply:
Honestly, I think we should publish PanelSide and all its component parts, since we are using it in props to define widgets. What do you think?

Reply:
@GerardasB or @raplemie should I move PanelSide and its constituent parts to appui-react and publish it?

Reply:
In the end, it is the same type than the abstract one, not sure its worth adding the Vertical/Horizontal and Individual types that are already just aliases to the strings. It makes the public API more complex than it needs to be, and the constitutents part will only be used to build the PanelSide...

I think I would keep the ""|""|""|"" version until we have more than a single use for it.

Reply:
Done!